code/modules/clothing/under/color.dm

```diff
@@ -30,6 +33,10 @@
 	item_state = "b_suit"
 	_color = "blue"
 	flags = ONESIZEFITSALL
+	species_fit = list("Vox")

```

Comment:
Because I'm an idiot and uploaded an old save for the file lmfao, oh God.

They're all supposed to look like the Grey jumpsuit's code block, with a species_fit before every sprite_sheets.

OOPS